var/spack/repos/builtin/packages/py-cdat-lite/package.py

```diff
@@ -0,0 +1,42 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+from spack import *
+
+
+class PyCdatLite(PythonPackage):
+    """Cdat-lite is a Python package for managing and analysing climate
+    science data. It is a subset of the Climate Data Analysis Tools (CDAT)
+    developed by PCMDI at Lawrence Livermore National Laboratory."""
+
+    homepage = "http://proj.badc.rl.ac.uk/cedaservices/wiki/CdatLite"
+    url      = "https://pypi.python.org/simple/cdat-lite/cdat-lite-6.0.1.tar.gz"
```

Comment:
Because that page isn't list-able. If you can find a page that lists all available versions and provides links to them, then you can add it as a `list_url`. In the long run, we need a better solution to this problem. I started work on a PyPIFetchStrategy to fix this but have been too busy to get it working the way I want.